frontend/src/scenes/urls.ts

```diff
@@ -7,12 +7,20 @@ export const urls = {
     notFound: () => '404',
     dashboards: () => '/dashboard',
     dashboard: (id: string | number) => `/dashboard/${id}`,
-    createAction: () => `/action`, // TODO: For consistency, this should be `/action/new`
-    action: (id: string | number) => `/action/${id}`,
-    actions: () => '/events/actions',
-    eventStats: () => '/events/stats',
-    eventPropertyStats: () => '/events/properties',
-    events: () => '/events',
+    // TODO: Deprecate below with `collaborations-taxonomy` FF
+    LEGACY_createAction: () => `/action`, // TODO: For consistency, this should be `/action/new`
+    LEGACY_action: (id: string | number) => `/action/${id}`,
+    LEGACY_actions: () => '/events/actions',
+    LEGACY_eventStats: () => '/events/stats',
+    LEGACY_eventPropertyStats: () => '/events/properties',
+    LEGACY_events: () => '/events',
+    // TODO: Deprecate above with `collaborations-taxonomy` FF
+    events: () => '/events_v2', // live events
+    taxonomy: () => '/taxonomy',
+    taxonomyEventStats: () => '/taxonomy/events/stats',
```

Comment:
Not sure how you're thinking about the new taxonomy page, but I would've assumed that stats are part of the new taxonomy and then you just have events and properties tabs (and maybe actions?)

Reply:
Yup that's completely right and partly my fault for being bad at naming. Tomorrow's taxonomy page will have two tabs called Events and Actions (Calculated Events in the future). Properties will be brought under Events tab (as discussed in https://github.com/PostHog/posthog/issues/7589) and stats for both will be incorporated into the taxonomy definitions UX.

I'll rename this from `taxonomyEventStats` to `taxonomyEvent` to communicate this better.

Reply:
That makes sense, but then I would assume `/taxonomy` page just loads the events tab with all that stuff? and then you just have an additional page for actions/calculated events